site2/website-next/contribute/documentation/contribution.md

```diff
@@ -0,0 +1,195 @@
+
+    :::
+    ```
+
+## Update reference docs
+
+If you want to update [Pulsar configuration docs](https://pulsar.apache.org/reference/#/latest/), pay attention to the doc source files.
+
+- Some docs are generated from code **automatically**. If you want to update the docs, you need to update the source code files.
+
+- Some configuration docs are updated **manually** using .md files.
+
+### Update Pulsar configuration docs
+
+<table>
+  <tr>
+   <td><strong>Components</strong>
+   </td>
+   <td><strong>Update where…</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Broker
+   </td>
+   <td>org.apache.pulsar.broker.ServiceConfiguration
+   </td>
+   <td rowspan="5" >These components are <strong>internal</strong>.
+    <p>These configuration docs are generated from code <strong>automatically</strong></p>.
+   </td>
+  </tr>
+  <tr>
+   <td>Client
+   </td>
+   <td>org.apache.pulsar.client.impl.conf.ClientConfigurationData
+   </td>
+  </tr>
+  <tr>
+   <td>WebSocket
+   </td>
+   <td>org.apache.pulsar.websocket.service.WebSocketProxyConfiguration
+   </td>
+  </tr>
+  <tr>
+   <td>Proxy
+   </td>
+   <td>org.apache.pulsar.proxy.server.ProxyConfiguration
+   </td>
+  </tr>
+  <tr>
+   <td>Standalone
+   </td>
+   <td>org.apache.pulsar.broker.ServiceConfiguration
+   </td>
+  </tr>
+  <tr>
+   <td>BookKeeper
+   </td>
+   <td><a href="https://github.com/apache/pulsar/blob/master/site2/docs/reference-configuration-bookkeeper.md">reference-configuration-bookkeeper.md</a>
+   </td>
+   <td rowspan="4" >These components are <strong>external</strong>.
+    <p>These configuration docs are updated <strong>manually</strong></p>.
+   </td>
+  </tr>
+  <tr>
+   <td>Log4j
+   </td>
+   <td><a href="https://github.com/apache/pulsar/blob/master/site2/docs/reference-configuration-log4j.md">reference-configuration-log4j.md</a>
+   </td>
+  </tr>
+  <tr>
+   <td>Log4j shell
+   </td>
+   <td><a href="https://github.com/apache/pulsar/blob/master/site2/docs/reference-configuration-log4j-shell.md">reference-configuration-log4j-shell.md</a>
+   </td>
+  </tr>
+  <tr>
+   <td>ZooKeeper
+   </td>
+   <td><a href="https://github.com/apache/pulsar/blob/master/site2/docs/reference-configuration-zookeeper.md">reference-configuration-zookeeper.md</a>
+   </td>
+  </tr>
+</table>
+
+### Update client configuration docs
+
+Pulsar Java configuration docs are generated from code automatically.
+
+| Components | Update where…                                                |
+| ---------- | ------------------------------------------------------------ |
+| Client     | org.apache.pulsar.client.impl.conf.ClientConfigurationData   |
+| Producer   | org.apache.pulsar.client.impl.conf.ProducerConfigurationData |
+| Consumer   | org.apache.pulsar.client.impl.conf.ConsumerConfigurationData |
+| Reader     | org.apache.pulsar.client.impl.conf.ReaderConfigurationData   |
+
+### Update CLI tool docs
```

Comment:
@tisonkun FYI: the workflow of this part is changed. I'll update it after this PR gets merged.

Reply:
Thank you! I notice that @codelipenghui has merged this patch. You can make the change and please inform me when it's ready :)